website/content/api-docs/system/inspect/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: api
+page_title: /sys/internal/inspect - HTTP API
+description: >-
+  This endpoint is intended to inspect a specific internal subsytem for debugging purposes. It can be accessed with a root token or sudo priveleges.
```

Comment:
```suggestion
  This endpoint is intended to inspect a specific internal subsystem for debugging purposes. It can be accessed with a root token or sudo privileges.
```